.gitignore

```diff
@@ -60,6 +60,8 @@ coverage.xml
 .hypothesis/
 .pytest_cache/
 cover/
+test/
```

Comment:
test/ is part of the repo.

Reply:
Oops, I apologise about that - seems like I had a partial execution of the tests that never got to cleanup and therefore created a large number of files within my test/ directory.